### PR TITLE
link to 'Docker Multi-stage builds'

### PR DESCRIPTION
**Pull request by pixel365**

### CODE REVIEW

File: src/data/roadmaps/docker/content/106-building-container-images/102-image-size-and-security.md [hunk 1]
```diff
@@ -73,3 +73,8 @@ When building container images, it's essential to be aware of both image size an
 - **Scan images for vulnerabilities:** Use tools like [Anchore](https://anchore.com/) or [Clair](https://github.com/quay/clair) to scan your images for vulnerabilities and fix them before deployment.
 
 By following these best practices, you'll be able to build more efficient and secure container images, leading to improved performance and a reduced risk of vulnerabilities in your applications.
+
```
Comment (dansholds) [inline src/data/roadmaps/docker/content/106-building-container-images/102-image-size-and-security.md:76]:
```suggestion
```

Comment (dansholds) [reply src/data/roadmaps/docker/content/106-building-container-images/102-image-size-and-security.md:76]: Nit: just one new line not two

File: src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md [hunk 2]
```diff
@@ -57,4 +57,4 @@ To build an image from the Dockerfile, use the `docker build` command, specifyin
 docker build -t my-image:tag .
 ```
 
-After running this command, Docker will execute each instruction in the Dockerfile, in order, creating a new layer for each.
\ No newline at end of file
+After running this command, Docker will execute each instruction in the Dockerfile, in order, creating a new layer for each.
```
Comment (dansholds) [inline src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md:60]: What happened here? 😂

**Review by dansholds — CHANGES_REQUESTED** [review]

File: src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md (hunk 2 above)
Comment (pixel365) [reply src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md:60]: > What happened here? 😂

Nothing, just an empty line was automatically added when saving when viewing the file :)

File: src/data/roadmaps/docker/content/106-building-container-images/102-image-size-and-security.md (hunk 1 above)
Comment (pixel365) [reply src/data/roadmaps/docker/content/106-building-container-images/102-image-size-and-security.md:76]: +

File: src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md (hunk 2 above)
Comment (pixel365) [reply src/data/roadmaps/docker/content/106-building-container-images/100-dockerfiles.md:60]: wposaegjhsn

**Review by dansholds — APPROVED** [review]

**Pull request merged**